apparmor.d/profiles-m-r/qbittorrent

```diff
@@ -110,10 +212,58 @@ profile qbittorrent @{exec_path} {
   /{usr/,}bin/qpdfview        rPx,
   /{usr/,}bin/ebook-viewer    rPx,
   /{usr/,}lib/firefox/firefox rPx,
+  /{usr/,}bin/nautilus        rPx,
 
-  # file_inherit
-  owner /dev/tty[0-9]* rw,
+  profile open {
+    include <abstractions/base>
+    include <abstractions/xdg-open>
+    include if exists <abstractions/ubuntu-unity7-base>
+
+    /{usr/,}bin/xdg-open mr,
+
+    # Allowed apps to open
+    /{usr/,}bin/spacefm         rPx,
+    /{usr/,}bin/smplayer        rPx,
+    /{usr/,}bin/vlc             rPx,
+    /{usr/,}bin/mpv             rPx,
+    /{usr/,}bin/geany           rPx,
+    /{usr/,}bin/viewnior       rPUx,
+    /{usr/,}bin/qpdfview        rPx,
+    /{usr/,}bin/ebook-viewer    rPx,
+    /{usr/,}lib/firefox/firefox rPx,
+
+    /{usr/,}bin/{ba,da,}sh        rix,
+    /{usr/,}bin/{g,m,}awk         rix,
+    /{usr/,}bin/readlink          rix,
+    /{usr/,}bin/basename          rix,
+    /{usr/,}bin/xfce4-mime-helper rix,
+
+    owner @{HOME}/ r,
+
+    owner @{run}/user/@{uid}/ r,
+
+    # file_inherit
+    owner @{MOUNTS}/*/torrent/** r,
+    owner @{MOUNTS}/*/torrent/**.[0-9a-f]*.parts rw,
+    owner "@{MOUNTS}/*/torrent/**.!qB" rw,
+
+    owner @{HOME}/.xsession-errors w,
 
+    dbus send
+      bus=session
+      path=/org/gtk/vfs/Daemon
+      interface=org.gtk.vfs.Daemon
+      member=ListMonitorImplementations,
+
+    dbus send
+      bus=session
+      path=/org/gnome/{Nautilus,Totem,gedit}
+      interface=org.freedesktop.Application
+      member=Open
+      peer=(name="org.gnome.{Nautilus,Totem,gedit}"),
+
+    include if exists <local/qbittorrent_open>
```

Comment:
Any better way for `local` child include?

Reply:
No really, I think it is fine like this.